CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # CHANGELOG
 
 ## Unreleased
+* Added throttled_queues variable to cumulus
```

Comment:
Since the PR is now going to a release branch, the changelog entry should be under the appropriate heading. Unreleased is for PRs to the master branch.